pkgs/by-name/taldir/package.nix

```
@@ -6,19 +6,19 @@
 }:
 buildGoModule {
   pname = "taldir";
-  version = "0-unstable-2024-02-18";
+  version = "0-unstable-2025-01-25";
```

Since we're using a stable release with the tag, this should correspond to that version (see [Nixpkgs docs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning)):

```suggestion
  version = "1.0.3";
```